resources/server_configuration.rb

```diff
@@ -104,65 +105,7 @@
-              mysqld_max_statement_time: new_resource.mysqld_max_statement_time,
-              mysqld_connect_timeout: new_resource.mysqld_connect_timeout,
-              mysqld_wait_timeout: new_resource.mysqld_wait_timeout,
-              mysqld_max_allowed_packet: new_resource.mysqld_max_allowed_packet,
-              mysqld_thread_cache_size: new_resource.mysqld_thread_cache_size,
-              mysqld_sort_buffer_size: new_resource.mysqld_sort_buffer_size,
-              mysqld_bulk_insert_buffer_size: new_resource.mysqld_bulk_insert_buffer_size,
-              mysqld_tmp_table_size: new_resource.mysqld_tmp_table_size,
-              mysqld_max_heap_table_size: new_resource.mysqld_max_heap_table_size,
-              mysqld_myisam_recover: new_resource.mysqld_myisam_recover,
-              mysqld_key_buffer_size: new_resource.mysqld_key_buffer_size,
-              mysqld_open_files_limit: new_resource.mysqld_open_files_limit,
-              mysqld_table_open_cache: new_resource.mysqld_table_open_cache,
-              mysqld_myisam_sort_buffer_size: new_resource.mysqld_myisam_sort_buffer_size,
-              mysqld_concurrent_insert: new_resource.mysqld_concurrent_insert,
-              mysqld_read_buffer_size: new_resource.mysqld_read_buffer_size,
-              mysqld_read_rnd_buffer_size: new_resource.mysqld_read_rnd_buffer_size,
-              mysqld_query_cache_limit: new_resource.mysqld_query_cache_limit,
-              mysqld_query_cache_size: new_resource.mysqld_query_cache_size,
-              mysqld_query_cache_type: new_resource.mysqld_query_cache_type,
-              mysqld_default_storage_engine: new_resource.mysqld_default_storage_engine,
-              mysqld_general_log_file: new_resource.mysqld_general_log_file,
-              mysqld_general_log: new_resource.mysqld_general_log,
-              mysqld_log_warnings: new_resource.mysqld_log_warnings,
-              mysqld_slow_query_log: new_resource.mysqld_slow_query_log,
-              mysqld_slow_query_log_file: new_resource.mysqld_slow_query_log_file,
-              mysqld_long_query_time: new_resource.mysqld_long_query_time,
-              mysqld_log_slow_rate_limit: new_resource.mysqld_log_slow_rate_limit,
-              mysqld_log_slow_verbosity: new_resource.mysqld_log_slow_verbosity,
-              mysqld_log_output: new_resource.mysqld_log_output,
-              mysqld_options: new_resource.mysqld_options,
-              mysqldump_quick: new_resource.mysqldump_quick,
-              mysqldump_quote_names: new_resource.mysqldump_quote_names,
-              mysqldump_max_allowed_packet: new_resource.mysqldump_max_allowed_packet,
-              mysqldump_options: new_resource.mysqldump_options,
-              isamchk_key_buffer: new_resource.isamchk_key_buffer,
-              isamchk_options: new_resource.isamchk_options,
-              extra_configuration_directory: new_resource.extra_configuration_directory
-             )
+    variables build_mycnf_variables_hash
```

Comment:
👎 this seems like unnecessary redirection and will decrease the readability of the cookbook.

Reply:
I remove that part of the code, following your recos